src/models/index.ts

```diff
@@ -1,8 +1,10 @@
-import { Organization as OrganizationModel } from "./Organization";
-import { OrganizationUser as OrganizationUserModel } from "./OrganizationUser";
-import { User as UserModel, UserRole as UserRoleEnum } from "./User";
+import { Organization } from "./Organization";
+import { OrganizationUser } from "./OrganizationUser";
+import { User, UserRole } from "./User";
 
-export const OrganizationUser = OrganizationUserModel;
-export const Organization = OrganizationModel;
-export const User = UserModel;
-export const UserRole = UserRoleEnum;
+export default {
+  Organization,
```

Comment:
what I don't understand is why you can't just do the following in the files that need to import the models:

```ts
import { Organization } from "./models/Organization";
import { OrganizationUser } from "./models/OrganizationUser";
// etc... 
```

Reply:
I already tried the solution you are offering, but unfortunately it doesn't work. The simple `import` is not enough in order to initialize the model, the model should be concretely used in order for its `init()` method to be called, that means after importing the model I should invoke a function of its.
I want to avoid it, because to me it looks more error-prone: one should always remember to import and use the associative models when dealing with associations between other models, even if the associative models are not directly needed in the code.

Reply:
I have the opposite opinion, by initializing the models as a side effect of importing them you re hiding what is really going on